src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,120 @@
+package duke.command;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Objects;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.TaskList;
+import duke.task.Todo;
+import duke.util.Storage;
+import duke.util.Ui;
+
+/**
+ * Represents a command for adding tasks to the task lists. It comprises of 3 types
+ * of task to be created, namely todo, deadline and event.
+ */
+public class AddCommand extends Command {
+    private int taskType;
+    private String taskDetails;
+
+    /**
+     * Constructor to create an instance of AddCommand.
+     *
+     * @param taskType Integer value representing the type of task, 0-TODO Task,
+     *                 1-DEADLINE Task, 2-EVENT Task
+     * @param taskDetails String representation of the task description
+     * @return instance of AddCommand
```

Comment:
I thought that constructor does not return anything in Java? No need for @return tag